cfpb/consumerfinance.gov
# Remove "Structured Data" panel and schema_json field

This PR removes the unused "Structured Data" panel and associated `schema_json` field.

---



## Removals

- Code related to "Structured Data" panel and `schema_json` field
- Associated tests

## How to test this PR

1. localhost
2. Check Configuration tab of various page types to confirm "Structured Data" no longer exists


## Checklist



  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance
